plugins/synaptics-vmm9/README.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Plugin: Synaptics VMM9xxxx
+---
+
+## Introduction
+
+This plugin updates the VMM9xxxx MST devices from Synaptics.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a packed binary file format.
+
+This plugin supports the following protocol ID:
+
+* `com.synaptics.mst`
```

Comment:
I kind of feel like this is a new protocol.

Reply:
`com.synaptics.mst.hid` or `com.synaptics.mst-hid` perhaps?

Reply:
Yeah mst hid

Reply:
the dot or the dash?

Reply:
Fixed.